app/lib/pdf_filler/nc_d400_pdf.rb

```diff
@@ -54,7 +54,8 @@ def hash_for_pdf
         y_d400wf_li20a_pg2_good: @xml_document.at('IncTaxWith')&.text,
         y_d400wf_li20b_pg2_good: @xml_document.at('IncTaxWithSpouse')&.text,
         y_d400wf_li23_pg2_good: @xml_document.at('NCTaxPaid')&.text,
-        y_d400wf_li25_pg2_good: @xml_document.at('RemainingPayment')&.text
+        y_d400wf_li25_pg2_good: @xml_document.at('RemainingPayment')&.text,
+        y_d400wf_li10a_good: @xml_document.at('NumChildrenAllowed')&.text
```

Comment:
why are all these suffixed with `_good`?

Reply:
yes it is provided to us and therefore i have no idea why `_good` 😅